torch/nn/script/activation.py

```diff
@@ -0,0 +1,344 @@
+import warnings
+import torch
+import torch.nn as nn
+from torch.nn.parameter import Parameter
+
+from ..modules.module import Module
+from .. import functional as F
+from torch.jit import ScriptModule
+
+
+# NB: We don't support inplace operation since script don't support it.
+
+
+class Threshold(ScriptModule):
+    __doc__ = nn.Threshold.__doc__
+    __constants__ = ['threshold', 'value', 'inplace']
+
+    def __init__(self, threshold, value, inplace=False):
+        super(Threshold, self).__init__()
+        self.threshold = threshold
+        self.value = value
+        self.inplace = inplace
+        # TODO: check in THNN (if inplace == True, then assert value <= threshold)
+
+    @torch.jit.script_method
+    def forward(self, input):
+        return F.threshold(input, self.threshold, self.value)
+
+    def extra_repr(self):
+        inplace_str = ', inplace' if self.inplace else ''
+        return 'threshold={}, value={}{}'.format(
+            self.threshold, self.value, inplace_str
+        )
+
+
+class ReLU(Threshold):
+    __doc__ = nn.ReLU.__doc__
+
+    def __init__(self, inplace=False):
+        super(ReLU, self).__init__(0, 0, inplace)
+
+    def extra_repr(self):
+        inplace_str = 'inplace' if self.inplace else ''
+        return inplace_str
+
+
+class RReLU(ScriptModule):
+    __doc__ = nn.RReLU.__doc__
+    __constants__ = ['lower', 'upper', 'inplace', 'training']
+
+    def __init__(self, lower=1. / 8, upper=1. / 3, inplace=False):
+        super(RReLU, self).__init__()
+        self.lower = lower
+        self.upper = upper
+        self.inplace = inplace
+
+    @torch.jit.script_method
+    def forward(self, input):
+        # NB: aten::rrelu(Tensor self, Scalar lower=<default>, Scalar upper=<default>, int training=<default>
+        # , Tensor generator=<default>) function signature has additional argument `generator`
+
+        return F.rrelu(input, self.lower, self.upper, self.training, None)
+
+    def extra_repr(self):
+        inplace_str = ', inplace' if self.inplace else ''
+        return 'lower={}, upper={}{}'.format(self.lower, self.upper, inplace_str)
```

Comment:
Can't we use multiple inheritance from the regular nn modules to get those methods implemented?